docs/pages/getting_started_access_data.mdx

```diff
@@ -0,0 +1,93 @@
+
+**Namespace Prefixes:**
+Each origin supports one or more *namespace prefixes*, which are analogous to the folders or directories from your computer that you use to organize files. Below is an example object:
+```bash
+/demo/testfile.txt
+```
+This is a simple example where the namespace prefix would be `/demo` and the object is named `/testfile.txt`.
+
+However, things can get a little tricky when you add a longer namespace prefix and/or a longer filepath within that namespace where the object lives. For example:
+```bash
+/ospool/uc-shared/public/OSG-Staff/validation/test.txt
+```
+In this example, the namespace prefix is `/ospool/uc-shared/public/`, and the actual object is named `/OSG-Staff/validation/test.txt`. While the namespace prefix always comes before the object name, it can get a little confusing once paths and names get longer.
+
+>**How to differentiate namespace prefix and object name?**
+To discover where the namespace prefix ends and the object name begins, there are a few options. One way is to access your federation's registry (i.e. [https://osdf-registry.osg-htc.org/view/registry/](https://osdf-registry.osg-htc.org/view/registry/)) which should list the namespaces available in the federation. Another option is also asking your federation administrator for how to access the registry. We are currently working to improve user experience exploring federations which includes better ways to differentiate namespace prefix and object name.
+
+**Federations:**
+Objects in Pelican belong to *federations*, which are aggregations of data that are exposed to other individuals in the federation. Each Pelican federation constitutes its own global namespace of objects and each object within a federation has its own path, much like files on a computer. Fetching any object from a federation requires at minimum three pieces of information: a URL indicating the object's federation, a namespace prefix, and the path to the object within that federation (there is the potential that some objects require access tokens as well, but more on that later). For example, the OSDF’s federation URL is https://osg-htc.org, an example namespace-prefix from the federation is:
+```bash
+/ospool/uc-shared/public
+```
+and the location of the object can be found at:
+```bash
+/OSG-Staff/validation/test.txt
+```
+
+Combining these three pieces of information, we say the the **object's URL** is:
+```bash
+https://osg-htc.org/ospool/uc-shared/public/OSG-Staff/validation/test.txt
+```
+
+## Your First Pelican Client Command
+Accessing objects from a Pelican federation requires the use of the Pelican client commands. Here we will be using the `pelican object get` command to get an object from a federation. This command is formatted like so:
```

Comment:
Technically you can't get objects from a pelican federation with anything that speaks HTTP, not just Pelican clients. Maybe rephrase?